agentless-sms-notifications/java/src/com/genesys/Main.java

```diff
@@ -0,0 +1,49 @@
+package com.genesys;
+
+import com.mypurecloud.sdk.v2.ApiClient;
+import com.mypurecloud.sdk.v2.api.ConversationsApi;
+import com.mypurecloud.sdk.v2.model.*;
+
+
+public class Main {
+    
+    public static void main(String[] args) {
+        try {
+            //Define your OAuth client credentials
+            final String clientId = "e07e5863-48ec-4b59-bf76-88e07dc33t98";
```

Comment:
Change these to the following to get the client id and secret from environment variables instead of being hardcoded.

```java
final String clientId = System.getenv("PURECLOUD_CLIENT_ID");
final String clientSecret = System.getenv("PURECLOUD_CLIENT_SECRET");
```

If that's a real client ID and secret, regenerate the secret immediately as it has been compromised; this page is publicly accessible on the internet.

Reply:
👍 updated